### ISSUE TITLE
Spinner no longer works in IE or Edge

### ISSUE THREAD

**Issue by waltonseymour** [palantir/blueprint#2816]
#### Environment

- __Package version(s)__:  3.3.0
- __Browser and OS versions__: Windows 10, IE11 / Edge 17.
#### Steps to reproduce

1. Visit https://blueprintjs.com/docs/#core/components/spinner
#### Actual behavior
Spinner does not animate

#### Expected behavior

Spinner should animate.

#### Possible solution
I assume this is related to #2477. Also important to note that the spinners work on 2.3.1

**Comment by giladgray** [comment]
@waltonseymour there is a callout on the spinner docs declaring that they do not work properly in IE11 due to compatibility issues, so that's expected:
![image](https://user-images.githubusercontent.com/464822/44365011-62c8e380-a47d-11e8-9693-1aa49f8c56c2.png)

that callout isn't even fully correct, as while IE11 _does_ support `animation` on SVG elements, it does not support `transform`, which is essential for smoothly rotating an element.

So in conclusion, the `<Spinner>` component is **completely incompatible with IE11**.

**Comment by giladgray** [comment]
ah seems the big change in #2477 was removing the wrapper `<div>` on which we _could_ set `transform: rotate()` for IE support. bummer.

**Comment by waltonseymour** [comment]
Is it possible to add a wrapping div for IE/Edge support?

**Comment by waltonseymour** [comment]
Don't mean to be a pain, but what's the vision behind this?
Are we no longer supporting IE/Edge as stated in https://blueprintjs.com/docs/#blueprint.browser-support?
If so, it might be helpful to have a broader discussion about IE support going forward, as well as update the documentation to call out that this component is completely unsupported in IE.

If we do still plan on supporting IE, then this would classify as a bug and could be resolved with some changes (which I'd be happy to contribute).

Thanks for your time on blueprint, just hoping to get more context here.

**Comment by giladgray** [comment]
@waltonseymour we definitely still support IE11. this was a significant oversight and I've just opened a PR to fix it.
